components/IscnUploadForm.vue

```diff
@@ -173,9 +179,10 @@ export default class UploadForm extends Vue {
   fileName: string = ''
   fileSize: number = 0
   fileType: string = ''
-  uploadStatus: string = '';
+  uploadStatus: string = ''
 
   isOpenFileInfoDialog = false
+  isTooLargeFile = false
```

Comment:
Can u try using `uploadStatus` with 'size-exceeded' instead of adding a new boolean

Reply:
I think it would be better to use boolean because we have to bind it with the v-model, also I found that `uploadStatus` is useless in this component, so I decided to remove `uploadStatus`.